### PR TITLE
Improvements(UI): Improved styling for global settings page and some minor bug fixes.

### LINKED ISSUE #6702: Redirection on docs page for Connectors from profiler tab is not proper.
**Affected module**
UI

**Describe the bug**
When clicked on "here" in profiler tab for entity details page, we get 404 as we redirect to /connectors.

**To Reproduce**
Screenshot 2022-08-12 at 10 00 17 AM
Screenshot 2022-08-12 at 10 00 33 AM

**Expected behavior**
It should redirect to /openmetadata/connectors
Screenshot 2022-08-12 at 10 00 50 AM

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

### PR DESCRIPTION
**Pull request by aniketkatkar97**
Closes #6702
### Describe your changes :

I worked on the improving the styling for : 
- [x] Left panel styling (stroke and shadow) (Update it as per the styling on other pages)
- [x] Add more margin b/w left panel and content 
- [x] Manage button width to be in 1:1 ratio
- [x] Change icon colour of selected section colour to primary colour and font weight semi-bold
- [x] Reduce delete button switch size
- [x] Maintain top margin
- [x] Fixed issue with redirection link to docs from profiler tab
#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">
Screenshot 2022-08-16 at 7 59 42 PM
Screenshot 2022-08-16 at 7 59 55 PM
Screenshot 2022-08-16 at 8 00 32 PM
Screenshot 2022-08-16 at 8 00 38 PM

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less [hunk 1]
```diff
@@ -11,30 +11,62 @@
  *  limitations under the License.
  */
 
-.global-setting-left-panel.ant-menu-inline {
-  box-shadow: 0px 2px 8px 0px rgba(0, 0, 0, 0.15);
-  width: 290px;
-  padding: 16px 0;
+.global-setting-left-panel.ant-menu-root.ant-menu-inline {
+  background: #ffffff;
+  border: 1px solid rgba(229, 231, 235, var(--tw-border-opacity));
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less:16]: Do we need an opacity variable here?

File: openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less [hunk 2]
```diff
@@ -11,30 +11,62 @@
...
     height: 30px;
     line-height: 30px;
     margin-top: 2px;
     padding: 0 16px !important;
   }
+
+  .ant-menu-item:hover,
+  .ant-menu-item-selected {
+    .side-panel-icons {
+      color: #7147e8;
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less:51]: Can we use variables for colour?

File: openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less (hunk 1 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less:16]: I have given this as per the mocks. Should I remove it?

**Review by Sachin-chaurasiya — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less (hunk 1 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less:16]: No, Can we use less variables then?

Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less:16]: were did you declare `--tw-border-opacity` variable? this is the tailwind style of CSS, I think we should directly provide number

File: openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less [hunk 3]
```diff
@@ -11,30 +11,62 @@
...
+      font-weight: 600;
+    }
+  }
+
   .ant-menu-item:not(:last-child) {
     margin-bottom: 0px;
   }
+
+  .ant-menu:not(.ant-menu-horizontal) .ant-menu-item-selected {
+    border-right: 1px solid rgba(229, 231, 235, var(--tw-border-opacity));
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less:66]: please declare a variable as `less` style if we are using common numbers

**Review by ShaileshParmar11 — APPROVED** [review]
miner comments otherwise looks good

File: openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less (hunk 1 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less]: Done

File: openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less (hunk 2 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less]: Done

File: openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less (hunk 3 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less]: Done

**Pull request merged**